README.md

```diff
@@ -34,10 +34,116 @@ or `git clone https://github.com/suhailshergill/extensible-effects.git`
 2. start `stack ghci` or `cabal repl`
 3. import some library modules as described in this section
 
-*examples are a work in progress and there will be some Quickstart module to go
-along the guide here*
+### Examples
 
-*examples...*
+In this section, a lot of examples are given. For technical details, see the
+next section.
+
+Note that most examples given here are very small. For them,
+using `Eff` monad is more complicated.
+The power of extensible effects lie in the fact that these comutations can be
+used to construct much more complicated programs by composing the little pieces
+shown here.
+
+All examples from this module can be found and tested in the module
```

Comment:
@power-fungus i am hesitant to have the same piece of code be in two places, especially when one of them doesn't go through compilation check (which makes it even more burdensome to maintain). we have a couple of options:

1. we simply hyperlink to the github location of the `Control.Eff.Quickstart` module here in the README

2. orthogonal to 1, we may also convert `Control.Eff.Quickstart` to a literal haskell file to improve readability and exposition if that's desired

Reply:
1. I would like for users to read the haddock-generated documentation instead of source-code. Indeed, the duplication is hard to maintain. Referring readers to the `Control.Eff.QuickStart` module without link should works fine in every context (hackage, stackage, github).
2. I perfer leaving it in `.hs` format, as this is more common. Other deviations from the "standard" would be more distracting for beginners.